openssl/src/ssl/mod.rs

```diff
@@ -774,12 +774,12 @@ impl SslContextBuilder {
     /// curve.
     ///
     /// Requires the `v102` feature and OpenSSL 1.0.2.
-    #[cfg(all(feature = "v102", ossl102))]
+    #[cfg(all(feature = "v102", any(ossl102, libressl)))]
```

Comment:
What would the feature be named? Wouldn't it be better to try to be as seamless as possible? (I probably don't realize what all the implications are)